wavedump/src/wavedump.c

```diff
@@ -1681,21 +1772,35 @@ int main(int argc, char *argv[])
     }
 
     signal(SIGINT, sigint_handler);
-
-    if (config_filename) {
-        printf("Opening Configuration File %s\n", config_filename);
-        f_ini = fopen(config_filename, "r");
-        if (f_ini == NULL) {
-            fprintf(stderr, "couldn't find configuration file '%s'\n", config_filename);
-            exit(1);
-        }
-        memset(&WDcfg, 0, sizeof(WDcfg));
-        ParseConfigFile(f_ini, &WDcfg);
-        fclose(f_ini);
+    
+    /* TODO: make methods that set all these settings.
+     * Make separate methods for the settings needed for
+     * 511 and SPE. */
+    WDcfg = set_default_settings();
+    
+    /* Set to enable external triggers, so we can trigger on the laser if we
+     * want. */
+    if (external) {
+        WDcfg.ExtTriggerMode = CAEN_DGTZ_TRGMODE_ACQ_ONLY;
+    } else {
+        WDcfg.ExtTriggerMode = CAEN_DGTZ_TRGMODE_DISABLED;
     }
 
     WDcfg.voltage = voltage;
     WDcfg.barcode = barcode;
+    
+    /* config file is an outdated method to program digitizer */
+    // if (config_filename) {
+    //     printf("Opening Configuration File %s\n", config_filename);
+    //     f_ini = fopen(config_filename, "r");
+    //     if (f_ini == NULL) {
+    //         fprintf(stderr, "couldn't find configuration file '%s'\n", config_filename);
+    //         exit(1);
+    //     }
+    //     memset(&WDcfg, 0, sizeof(WDcfg));
+    //     ParseConfigFile(f_ini, &WDcfg);
+    //     fclose(f_ini);
+    // }

```

Comment:
Although we aren't using it would be nice to be able to have this option in the future so I would leave this uncommented.